lib/awful/tag.lua

```diff
@@ -284,7 +283,7 @@ function tag.object.delete(self, fallback_tag, force)
 
     -- Abort if we would have un-tagged clients.
     local clients = self:clients()
-    if ( #clients > 0 and ntags <= 1 ) or fallback_tag == nil then return end
+    if #clients > 0 and fallback_tag == nil then return false end
```

Comment:
The old logic was "don't allow deleting the last tag of a screen". I think that's quite a good idea since screens without tags will surely cause many problems.

How about making this also depend on the `force` argument to this function?
So:
```lua
if fallback_tag == nil then return false end
local clients = self:clients()
if #clients > 0 and ntags <= 1 and not force then return false end
```
(By the way, did you change the `or` in the old code into an `and` on purpose?)